src/libs/mrubyWriterConnector.ts

```diff
@@ -492,4 +496,26 @@ export class MrubyWriterConnector {
 
     return Success.value(line);
   }
+  async verify(code: Uint8Array) {
+    const crc = calculateCrc8(code);
+    const verifyRes = await this.sendCommand("verify");
+    console.log("verifyRes:", verifyRes);
+    if (verifyRes.isFailure()) return verifyRes;
+
+    const crcRes = verifyRes.value.match(/^\+OK (?<hash>[0-9a-zA-Z]+)\r?\n$/)
+      ?.groups?.hash;
+    console.log("crcRes", crcRes);
+    if (!crcRes) {
+      this.handleText("\r\n\u001b[31m Verify failed. \r\n");
```

Comment:
```suggestion
      this.handleText("\r\n\u001b[31m failed to verify. \r\n");
```
文法は正しく、書式は他のものと合わせてくれると嬉しいです。